ToolsUtilities/FileManager.cs

```diff
@@ -21,16 +21,29 @@ public static partial class FileManager
         public const char DefaultSlash = '\\';
         #region Fields
 
-        public static string ExeLocation =>
+        public static string ExeLocation
+        {
+            get
+            {
 #if UWP
-            "./";
+            return "./";
 #elif ANDROID || IOS
-            Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location).ToLower().Replace("/", "\\") + "\\";
+            return Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location).ToLower().Replace("/", "\\") + "\\";
 #else
-            Path.GetDirectoryName(AppContext.BaseDirectory)
-                .Replace('/', Path.DirectorySeparatorChar)
-                .Replace('\\', Path.DirectorySeparatorChar) + Path.DirectorySeparatorChar;
+                try
+                {
+                    return Path.GetDirectoryName(AppContext.BaseDirectory)
+                            .Replace('/', Path.DirectorySeparatorChar)
+                            .Replace('\\', Path.DirectorySeparatorChar) + Path.DirectorySeparatorChar;
+                }
+                catch 
+                {
+                    // WASM ?
+                    return "./";
```

Comment:
AppContext.BaseDirectory throws in WASM, and I changed ExeLocation to "./".
I am not sure if that is the correct fix here.

Reply:
I think that's okay, unless we can get a proper #if

Reply:
Actually it was Path.GetDirectoryName() that was failing.
Now I skip it when the value is "/", and let the rest of the method to do, whatever it does.
This also fixed the issue we had with /Content/Content/ path!